plugins/redux/src/components/Counter/index.tsx

```diff
@@ -3,21 +3,40 @@ import { useDispatch, useSelector } from "react-redux";
 
 import { increase, decrease } from "@redux/actions";
 import { IState } from "@redux/istate";
+import styles from "./index.module.css";
 
 export default function Counter() {
   const dispatch = useDispatch();
   const count = useSelector((state: IState) => state.counter.count);
 
   return (
-    <>
-      <h2>Counter</h2>
-      <button type="button" onClick={() => dispatch(increase())}>
-        +
-      </button>
-      <strong style={{ margin: 10 }}>{count}</strong>
-      <button type="button" onClick={() => dispatch(decrease())}>
-        -
-      </button>
-    </>
+    <div className={styles["box"]}>
+      <div>
+        <h2>Counter</h2>
+        <button
+          className={[styles.button, styles["button--increase"]].join(" ")}
+          type="button"
+          onClick={() => dispatch(increase())}
+        >
+          +
+        </button>
+        <span className={styles["count"]}>{count}</span>
+        <button
+          className={[styles.button, styles["button--decrease"]].join(" ")}
+          type="button"
+          onClick={() => dispatch(decrease())}
+        >
+          -
+        </button>
+      </div>
+
+      <a
+        className={styles["link"]}
+        href="https://react-redux.js.org/"
+        target="_blank"
+      >
+        Go to redux documentation.
```

Comment:
```suggestion
        Go To Documentation
```
tüm componentlerin aynı olması için bu şekilde yaparsak daha iyi